win32ss/drivers/miniport/xboxvmp/xboxdisp.inf

```diff
@@ -49,9 +49,9 @@ HKR,,InstalledDisplayDrivers,0x00010000,framebuf
 [Strings]
 ; Non-localizable
 ReactOS = "ReactOS Project"
+nVidiaMfg = "nVidia"
```

Comment:
`nVidia` is something else than `nVidia Corporation`

Reply:
No, and we use the shorter version in some other inf files already.